gammapy/utils/testing.py

```diff
@@ -1,23 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """Utils to create scripts and command-line tools"""
 from __future__ import print_function, division
-from numpy.testing import assert_allclose
-from astropy.units import Quantity
-from astropy.coordinates import Angle
 
-__all__ = ['assert_quantity',
-           ]
-
-
-def assert_quantity(actual, desired, *args, **kwargs):
-    """Assert value and unit for astropy Quantity.
-
-    Calls `numpy.testing.assert_allclose`.
-    """
-    if not isinstance(actual, Quantity):
-        raise ValueError("actual must be a Quantity object.")
-    if not isinstance(desired, Quantity):
-        raise ValueError("desired must be a Quantity object.")
-
-    assert actual.unit == desired.unit
-    assert_allclose(actual, desired, *args, **kwargs)
+__all__ = []
```

Comment:
Just remove it for now ... we can always add it back later if we need it.
There might also be a Sphinx automodapi directive you have to remove in the docs.
And you can use the Github full-text search to  make sure you've replaced all the callers (test coverage in Gammapy is so bad that you can't rely on tests passing)